### PR TITLE
Ignore records that fail to serialize in the buffered output

### PR DESCRIPTION
**Pull request by IvanVergiliev**
Yajl.dump cannot handle records with NaN values in them; however, a
large number of tools produce such records (including, AFAICT, the
fluentd node logger). This change handles the exception and ignores the
message.

If the message is not ignored, basically everything breaks. Fluentd
keeps trying to resend the buffer that contains the message, always
failing because the message hasn't changed and still cannot be
serialized; an ever increasing number of buffer files are created, and,
eventually, the size limit is reached and fluent starts dropping data.
TL;DR currently the buffered plugin has no way of dealing with non-transient
errors beyond manually deleting the offending buffer files which is
far from ideal.

### CODE REVIEW

**Review by roiravhon — APPROVED** [review]

**Comment by roiravhon** [comment]
Pushed as part of version 0.0.11
